### ISSUE TITLE
Please give a trivial usage example

### ISSUE THREAD

**Issue by nirgal** [stacktracejs/stacktrace.js#139]
Hi guys, I don't quite know what you are smoking with nodeis and unsiversal support and so no and so forth, but I notice your project now have over 20 files, and no proper example.

I would like to have something like:
```
<html>
<script src="/javascript/stacktrace.js">
<script>
try {
   ...
} exception (e) {
   alert(stacktrace(e));
}
</script>
```

And I would very much appreciate a javascript file that one could actually use in a browser (remember?), like what was at http://www.eriwen.com/javascript/js-stack-trace/

**Comment by oliversalzburg** [comment]
The files you're looking for are in https://github.com/stacktracejs/stacktrace.js/tree/master/dist. For everything else, please see the [documentation](http://www.stacktracejs.com/#!/docs/stacktrace-js) and the [migration guide](http://www.stacktracejs.com/#!/docs/v0-migration-guide).

**Comment by nirgal** [comment]
The migration guide says you are using promises, and points at http://caniuse.com/#search=promises that says that it doesn't work on 2 third of browsers? Oo

I'm lost, the README says IE6+ is supported. Is the README outdated?

Am I missing something?

You realize that when you have a web site, you want it to work for all people, and 33% of users being thrown away is unacceptable, right? Are these functions still usable in a website or not?

**Comment by oliversalzburg** [comment]
Promises aren't *natively* supported in many browsers. Which is why stacktrace.js comes with a polyfill version, which allows it to use promises on all browsers.

**Comment by nirgal** [comment]
How can I get or make that polyfill version ?

I am not much interested in an uglyfied version, I need the sources.


I noticed file `dist/stacktrace-with-polyfills.min.js.map` containing
`"sources":["es6-promise.js","json3.js","polyfills.js","stacktrace-gps.min.js","stack-generator.js","error-stack-parser.js","stacktrace.js"]`
Am I correct in assuming that you need to include these 7 files in that order to make the library work?
If so, where can I find them?
I can see the source of polyfills.js and stacktrack.js, but not of the other files.

**Comment by oliversalzburg** [comment]
The repository currently does not contain an unminified version of the polyfill variant. You don't need to include additional files, they are contained in the polyfill variant. If you want to retrieve them anyway, you can do so by running `npm install` to pull in all dependencies.

**Comment by nirgal** [comment]
But that so called polyfill variant, the only one that can be used in a web site, has no source.
And if I want the source, I need to install node ? This is really not cool.

I found out there still is a "stable" branch in this repository. I tried it and it works perfectly. The code is small, self contained. It also works with internet explorer using the callee.caller trick, that I suspect the so called "master" version dropped. Please consider that people needing backtrace are quite desperate and need to understand what the hell is going on in corner cases, like people running their code in palaeolithic browsers. If someone only needed the stack property of exceptions, well, one would use it directly!

A big thank you to people who contributed to stable branch!

It's a pity to see such a useful and nice library being slaughtered that way. :(

**Comment by oliversalzburg** [comment]
So, you're fine with the old version then? Great! I guess, that means this ticket can be closed. 

We are really very sorry that you don't like the new version and we would love nothing more than being able to convince you of its benefits. We understand that it's an unbearable burden to ask you to install Node on your computer to build a version of the file that you want. If only I could find a way to make up the pain we have caused you :(

Well, here, have a :cake:, maybe that's a start. :heart: :kiss:

**Comment by nirgal** [comment]
Have fun if you like, but it would be really great to:

- Provide an open source version that can be used in browsers, just like the stable branch does.

- Restore support for internet explorer and other browser that don't support exception.stack property, just like the stable branch does.


You want the lib in universal node whatever, ok. But when the library is no longer usable for simple cases, it's a problem, in my opinion.

**Comment by eriwen** [comment]
@nirgal It sounds like all you want is one JS file to include on a web page and one simple method call to work in IE6+. It's still as easy as ever. Just:

Download [stacktrace.js with bundled dependencies](https://github.com/stacktracejs/stacktrace.js/releases/download/1.0.1/stacktrace-with-polyfills.min.js) and include it. Then write:

```js
StackTrace.get().then(function(stack) {
  console.log(stack);
});
```
wherever you want. We had to make v1.0 asynchronous because browsers have deprecated synchronous ajax requests. BTW, the 0.x version on stable doesn't always work on the latest Firefox because of the way we did stack parsing.

If you're willing to explain a bit more about your goal with stacktrace.js, we can advise further.
